incubating/pagerduty-alert/lib/pagerduty_alert.py

```diff
@@ -27,13 +27,14 @@ def main():
             },
             'assignments': [
```

Comment:
The if will need to be at assignments level.  Once assignments is in the payload the assignee is required.  Please move your conditional block up to that level.

https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE0MA-create-an-incident

Reply:
Block moved out and tested outside pager-duty plugin